EIPS/eip-1202.md

```diff
@@ -113,32 +99,6 @@ The third consideration is non-trivialness. Some voting applications assume ***a
  
 The fourth consideration is potential abuse. When voting is standardized and put on contract, it is possible to write another contract that rewards a voter to vote in a certain way. It creates potential issues of bribery and conflict of interest abuse that is previously hard to implement.
 
-
-## Bibliography
-### Related EIPs
- - [EIP-20: ERC-20 Token Standard (a.k.a. ERC-20)](./eip-20.md)
- - [EIP-165: Standard Interface Detection](./eip-165.md)
- - [EIP-721: Non-Fungible Token Standard(a.k.a. ERC-721)](./eip-721.md)
- - [EIP-735: ERC: Claim Holder](https://github.com/ethereum/EIPs/issues/735)
- - [EIP-780: ERC: Ethereum Claims Registry](https://github.com/ethereum/EIPs/issues/780)
- - [EIP-777: A New Advanced Token Standard](./eip-777.md)
- - [EIP-897: ERC DelegateProxy](./eip-897.md)
- - [EIP-1155: Crypto Item Standard](./eip-1155.md)
- - [EIP-1178: Multi-class Token Standard](./eip-1178.md)
- - [EIP-1167: Minimal Proxy Contract](./eip-1167.md)
- - [EIP-1203: Multi-class Token Standard(ERC-20 Extension)](./eip-1203.md)
-
-### Worthnoting Projects
- - [Ethereum DAO: How to build a DEMOCRACY on the blockchain](https://www.ethereum.org/dao)
- - [Carbon Vote](http://carbonvote.com/)
- - [Paper: A Smart Contract for Boardroom Voting with Maximum Voter Privacy](https://eprint.iacr.org/2017/110.pdf) - *Suggested by @aodhgan*
- - [Private Voting for TCR](https://blog.enigma.co/private-voting-for-tcrs-with-enigma-b441b5d4fa7b)
- - [Consensus/PLCR implementation](https://github.com/ConsenSys/PLCRVoting)
- - [Aragon Voting App](https://wiki.aragon.org/dev/apps/voting/)
-
-## Acknowledgement
-We appreciate Ansley, Andrew, Fred from Enigma, Fan and Raullen from IoTex for sharing us their use cases. we also appreciate the valuable input for designing an EIP from distinguished community members including: @frozeman, @fulldecent, @bingen, @aodhgan.
-
 ## Work Directory
 The drafting and revision of EIP-1202 is conducted at [GitHub/xinbenlv/eip-1202](https://github.com/xinbenlv/eip-1202)
```

Comment:
@lightclient how do you feel about this section? 😬

Reply:
EIPs are specifications, not a place for storing history, lists of tools, links, etc.  A good specification is short and concise without a bunch of space wasted on stuff unnecessary for implementing the something that is spec compliant.

Reply:
Got it. Makes sense!

Reply:
@MicahZoltu with the same argument, I wonder if we still want to keep "Rationale" in the standard.